database/config.js

```diff
@@ -1,16 +1,149 @@
+  let roomFeatureCopy8 = faker.lorem.sentence(2);
+
+  let roomTypeIcon1 = faker.random.number(3);
+  let roomTypeCopy1 = faker.lorem.sentence(2);
+  let roomTypeIcon2 = faker.random.number(3);
+  let roomTypeCopy2 = faker.lorem.sentence(2);
+  let roomTypeIcon3 = faker.random.number(3);
+  let roomTypeCopy3 = faker.lorem.sentence(2);
+  let roomTypeIcon4 = faker.random.number(3);
+  let roomTypeCopy4 = faker.lorem.sentence(2);
+
+  let hotelClass = faker.finance.amount(0, 5, 1);
+  let hotelStyle = faker.lorem.sentence(2);
+
+  let hotelWebsite = faker.internet.url();
+  let hotelDeals = faker.internet.url();
+
+  let params = [
+    average,
+    location,
+    cleanliness,
+    service,
+    value,
+    certOfExcellence,
+    greenLeaders,
+    aboutCopy,
+    amenityIcon1,
+    amenityCopy1,
+    amenityIcon2,
+    amenityCopy2,
+    amenityIcon3,
+    amenityCopy3,
+    amenityIcon4,
+    amenityCopy4,
+    amenityIcon5,
+    amenityCopy5,
+    amenityIcon6,
+    amenityCopy6,
+    amenityIcon7,
+    amenityCopy7,
+    amenityIcon8,
+    amenityCopy8,
+    roomFeatureIcon1,
+    roomFeatureCopy1,
+    roomFeatureIcon2,
+    roomFeatureCopy2,
+    roomFeatureIcon3,
+    roomFeatureCopy3,
+    roomFeatureIcon4,
+    roomFeatureCopy4,
+    roomFeatureIcon5,
+    roomFeatureCopy5,
+    roomFeatureIcon6,
+    roomFeatureCopy6,
+    roomFeatureIcon7,
+    roomFeatureCopy7,
+    roomFeatureIcon8,
+    roomFeatureCopy8,
+    roomTypeIcon1,
+    roomTypeCopy1,
+    roomTypeIcon2,
+    roomTypeCopy2,
+    roomTypeIcon3,
+    roomTypeCopy3,
+    roomTypeIcon4,
+    roomTypeCopy4,
+    hotelClass,
+    hotelStyle,
+    hotelWebsite,
+    hotelDeals
+  ];
+
+  connection.query('INSERT INTO about (average, location, cleanliness, service, value, certOfExcellence, greenLeaders, aboutCopy, amenityIcon1, amenityCopy1, amenityIcon2, amenityCopy2, amenityIcon3, amenityCopy3, amenityIcon4, amenityCopy4, amenityIcon5, amenityCopy5, amenityIcon6, amenityCopy6, amenityIcon7, amenityCopy7, amenityIcon8, amenityCopy8, roomFeatureIcon1, roomFeatureCopy1, roomFeatureIcon2, roomFeatureCopy2, roomFeatureIcon3, roomFeatureCopy3, roomFeatureIcon4, roomFeatureCopy4, roomFeatureIcon5, roomFeatureCopy5, roomFeatureIcon6, roomFeatureCopy6, roomFeatureIcon7, roomFeatureCopy7, roomFeatureIcon8, roomFeatureCopy8, roomTypeIcon1, roomTypeCopy1, roomTypeIcon2, roomTypeCopy2, roomTypeIcon3, roomTypeCopy3, roomTypeIcon4, roomTypeCopy4, hotelClass, hotelStyle, hotelWebsite, hotelDeals) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', params, (err, data) => {
+    if (err) {
+      console.log('error at database/config.js INSERT INTO', err);
+    } else {
+      counter++;
+      console.log(counter, 'data has been inserted');
+      if (counter < 100) {
+        seed();
+      }
+    }
+  });
+};
+
+seed();
+
+module.exports = seed;
+//
+
+
+
+// AMAZON
```

Comment:
what does this Amazon reference?

Reply:
below that is Jordan's mysql.createConnection settings for AWS. just thought i'd leave it in there for future reference.

Reply:
indubitably